src/configuration/configuration-base.ts

```diff
@@ -116,44 +135,89 @@ export default class Configuration {
         return cloneDeep(this);
     }
 
-    public get filePath (): string | null {
+    public get filePath (): string | undefined {
         return this._filePath;
     }
 
+    public get defaultPaths (): string[] | undefined {
+        return this._defaultPaths;
+    }
+
     public async _load (): Promise<null | object> {
-        if (!this.filePath)
+        if (!this.defaultPaths?.length)
             return null;
 
-        if (!await this._isConfigurationFileExists())
-            return null;
+        const configs = await Promise.all(this.defaultPaths.map(async filePath => {
+            if (!await this._isConfigurationFileExists(filePath))
+                return { filePath, options: null };
+
+            let options = null as object | null;
+
+            if (this._isJSConfiguration(filePath))
+                options = this._readJsConfigurationFileContent(filePath);
+            else {
+                const configurationFileContent = await this._readConfigurationFileContent(filePath);
+
+                if (configurationFileContent)
+                    options = this._parseConfigurationFileContent(configurationFileContent);
+            }
+
+            return { filePath, options };
+        }));
 
-        const configurationFileContent = await this._readConfigurationFileContent();
+        const existedConfigs = configs.filter(config => !!config.options);
 
-        if (!configurationFileContent)
+        if (!existedConfigs.length)
             return null;
+        else if (existedConfigs.length > 1) {
+            const priorityList = configs.map((item, index) => `${index + 1}. ${item.filePath}`);
```

Comment:
For better reading please combine the `priorityList` calculation and result string into single function:

```js
else if (existedConfigs.length > 1) { 
   const configPriorityListStr = this._getConfigPriorityListString (configs);
  
Configuration._showConsoleWarning(renderTemplate(WARNING_MESSAGES.multipleConfigurationFilesFound, configPriorityListStr));
  
}
```

Reply:
Also need to update the `WARNING_MESSAGES.multipleConfigurationFilesFound` message and say what specific file will be used in this test run.